<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>img_magnified_view</class>
 <widget class="QWidget" name="img_magnified_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* -----------------------------------------
   MAGNIFIED IMAGE VIEW PAGE
----------------------------------------- */
QWidget#img_magnified_view {
    background-color: rgba(30, 30, 40, 0.6); /* semi-transparent dark */
    border-radius: 16px;
    padding: 12px;
    border: 1px solid rgba(255, 255, 255, 0.1);
}

/* -----------------------------------------
   IMAGE WIDGET (left side)
----------------------------------------- */
QWidget#img_widget {
    background-color: rgba(255, 255, 255, 0.12);  /* translucent */
    border-top-left-radius: 16px;
    border-bottom-left-radius: 16px;
    border-top-right-radius: 0px;
    border-bottom-right-radius: 0px;
    border: 1px solid rgba(255, 255, 255, 0.2);
    padding: 8px;
    transition: all 0.2s ease;
}

QWidget#img_widget:hover {
    background-color: rgba(255, 255, 255, 0.2);
    border: 1px solid rgba(255, 255, 255, 0.35);
    box-shadow: 0 8px 24px rgba(180, 150, 220, 0.4);
}

/* -----------------------------------------
   IMAGE LABEL
----------------------------------------- */
QLabel#img_lbl {
    border-radius: 12px;
    background-color: rgba(0,0,0,0.1);
}

/* -----------------------------------------
   INFO WIDGET (right side)
----------------------------------------- */
QWidget#info_widget {
    background-color: rgba(255, 255, 255, 0.12);  /* translucent */
    border-top-left-radius: 0px;
    border-bottom-left-radius: 0px;
    border-top-right-radius: 16px;
    border-bottom-right-radius: 16px;
    border: 1px solid rgba(255, 255, 255, 0.2);
    padding: 12px;
    transition: all 0.2s ease;
}

QWidget#info_widget:hover {
    background-color: rgba(255, 255, 255, 0.2);
    border: 1px solid rgba(255, 255, 255, 0.35);
    box-shadow: 0 8px 24px rgba(180, 150, 220, 0.4);
}

/* -----------------------------------------
   INFO LABELS
----------------------------------------- */
QLabel#uploaded_by_lbl,
QLabel#uploaded_at_lbl,
QLabel#like_counter_lbl {
    color: #F5F0FF;
    font-weight: 500;
    font-size: 14px;
    padding: 2px 0;
}

/* -----------------------------------------
   LIKE BUTTON
----------------------------------------- */
QPushButton#like_btn {
    background-color: rgba(255, 255, 255, 0.12);
    color: #F5F0FF;
    border: 1px solid rgba(255, 255, 255, 0.25);
    border-radius: 6px;
    padding: 6px 12px;
    font-weight: 600;
    font-size: 14px;
    transition: all 0.2s ease;
    min-height: 36px;
}

QPushButton#like_btn:hover {
    background-color: rgba(255, 255, 255, 0.2);
    border: 1px solid rgba(255, 255, 255, 0.4);
    box-shadow: 0 6px 16px rgba(180, 150, 220, 0.4);
}

/* -----------------------------------------
   DOWNLOAD BUTTON
----------------------------------------- */
QPushButton#download_btn {
    background-color: rgba(255, 255, 255, 0.12);
    color: #F5F0FF;
    border: 1px solid rgba(255, 255, 255, 0.25);
    border-radius: 6px;
    padding: 6px 12px;
    font-weight: 600;
    font-size: 14px;
    transition: all 0.2s ease;
    min-height: 36px;
}

QPushButton#download_btn:hover {
    background-color: rgba(255, 255, 255, 0.2);
    border: 1px solid rgba(255, 255, 255, 0.4);
    box-shadow: 0 6px 16px rgba(180, 150, 220, 0.4);
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
   <item>
    <widget class="QWidget" name="img_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="image_lbl">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="info_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="uploaded_by_lbl">
        <property name="text">
         <string>uploaded by</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="uploaded_at_lbl">
        <property name="text">
         <string>uploaded at</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="like_counter_lbl">
          <property name="text">
           <string>likes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="like_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>like</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="download_btn">
        <property name="text">
         <string>download</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
